data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -1467,6 +1467,12 @@
       "actionid": 90,
       "actioncode": "",
       "tenantId": "pg"
+    },
+    {
```

Comment:
@chetankolte1  update the role actions to MUKTA_ADMIN and WORK_ORDER_CREATOR

Reply:
superuser can be removed

Reply:
Removed superuser and kept MUKTA_ADMIN and WORK_ORDER_CREATOR